src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -342,6 +342,7 @@ def mock_merge(*args, **kwargs):  # pylint: disable=unused-argument
         'strategy_dataflow_tracing': 0,
         'strategy_fork': 0,
         'strategy_mutator_plugin': 0,
+        'strategy_mutator_plugin_radamsa': 0,
```

Comment:
do we also need to update `src/python/metrics/fuzzer_stats_schema.py`? //cc @oliverchang

Reply:
Yep, we need to add a column there.

Reply:
Another idea: rather than create a column for every single plugin we have, how about having a column which has a string value instead, with the value being the plugin name?

Reply:
For now I just added another column. Let me know if I didn't add it right.

Reply:
Looks right to me, thanks!